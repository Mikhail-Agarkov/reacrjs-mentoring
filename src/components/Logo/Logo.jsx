import React from 'react';
import styles from './logo.module.scss';
import {Link} from 'react-router-dom';

const Logo = () => (
    <Link to='/' className={styles.link}>
        <span className={styles.logo}><span className={styles.bold}>netflix</span>roulette</span>
    </Link>
);

export default Logo;
