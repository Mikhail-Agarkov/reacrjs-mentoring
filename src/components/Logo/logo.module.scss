.logo {
  color: #F65261;
  font-family: Montserrat, sans-serif;
  font-size: 20px;
  font-weight: 500;
  line-height: 24px;
}

.bold {
  font-weight: 900;
}