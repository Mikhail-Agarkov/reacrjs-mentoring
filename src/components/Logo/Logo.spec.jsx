import React from 'react';
import {shallow} from 'enzyme';

import Logo from './Logo';

describe('Logo', () => {
    it('should render logotype of the site correctly', () => {
        const component = shallow(<Logo/>);
        expect(component).toMatchSnapshot();
    })
});