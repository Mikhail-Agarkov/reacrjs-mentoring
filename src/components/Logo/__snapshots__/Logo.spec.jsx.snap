// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render logotype of the site correctly 1`] = `
<Fragment>
  <span
    className="logo"
  >
    <span
      className="bold"
    >
      netflix
    </span>
    roulette
  </span>
</Fragment>
`;
